### DISCUSSION TITLE
Please help me with error: INVALID_AUTH_KEY

### DISCUSSION THREAD

**Discussion by Prosperlong** [aptos-labs/aptos-developer-discussions#535] [Questions] [answered]
### Describe your question in detail.

Hi all. I have an old aptos wallet, i havent used it for a long time. Yesterday i received some $apt, then it accidentally rotate the authenticate key, so now i cannot send the money to another wallets. It show error INVALID_AUTH_KEY (this happen to all wallet app i used: pontem, petra, trust. Even when i used the Aptos cli too
Please help me
![C1586074-07A6-4B0A-B64F-34054BD82960](https://github.com/user-attachments/assets/1527578b-450f-44bd-9d1f-e339987a86b8)
![D57FB32B-7CD9-4026-B963-1644BFEC83BD](https://github.com/user-attachments/assets/bbe120e8-d760-4871-b862-c0e5e11c51e3)

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by 0xbrw** [comment]
CC @bv-aptos who might have more context with petra wallet

**Comment by adeshkhari** [comment]
Hi bro

**Reply by bv-aptos** [reply, marked as answer]
INVALID_AUTH_KEY is typically what you see after the account has been rotated.  Seeing INVALID_AUTH_KEY means that you no longer have access to your wallet @adeshkhari cc @bwdotcom.

~~Can you share your full wallet address?~~ In looking at the explorer though, it appears your account has not been rotated. Taking a deeper look.

**Reply by Prosperlong** [reply]
Dear sir, my wallet is : 0x464b278beeeb9fb456c90820079210ee7a7559ea13d946ccfd811f1765b5b34b. Please help me take a look @bv-aptos

**Comment by GhostWalker562** [comment]
A bad key rotation was conducted to a Keyless account.

https://explorer.aptoslabs.com/txn/1945156172?network=mainnet

The scheme of the `0xf6e01cfd98024ff05c2ebeb13fd8ea3fb5e5ca1c2d6263c1c7a5ac4d69966787` public key should be `single_key` instead of `ed25519`. Unfortunately those funds are no longer recoverable.

**Reply by adeshkhari** [reply]
Brother same problem is my wallet

**Reply by Prosperlong** [reply]
Dear sir that’s the receive wallet. Mine is 0x464b278beeeb9fb456c90820079210ee7a7559ea13d946ccfd811f1765b5b34b @GhostWalker562 thank you

**Reply by GhostWalker562** [reply]
Didn’t realize that. I misread the values. Your wallet was rotated and you don’t have access to it anymore.

It was rotated at this transaction: https://explorer.aptoslabs.com/txn/1945156172?network=mainnet

Please refer back to this: https://github.com/aptos-labs/aptos-developer-discussions/discussions/535#discussioncomment-11351975

**Reply by adeshkhari** [reply]
Sir solution reply

**Reply by adeshkhari** [reply]
> Dear sir that’s the receive wallet. Mine is 0x464b278beeeb9fb456c90820079210ee7a7559ea13d946ccfd811f1765b5b34b @GhostWalker562 thank you

Hi brother

**Reply by Prosperlong** [reply]
> Didn’t realize that. I misread the values. Your wallet was rotated and you don’t have access to it anymore.
> 
> It was rotated at this transaction: https://explorer.aptoslabs.com/txn/1945156172?network=mainnet
> 
> Please refer back to this: [#535 (reply in thread)](https://github.com/aptos-labs/aptos-developer-discussions/discussions/535#discussioncomment-11351975)

So there is no solution sir? That wallet is rotated without my permission. I sent money to that then it automatically rotated @GhostWalker562  @bv-aptos

**Comment by adeshkhari** [comment]
Brother same problem my wallet

**Comment by adeshkhari** [comment]
Hi brother same problem with me 
My wallet invalid auth key rotated this my wallet address 0xd33d3b71aa63b6c1f0ba249f5122b1fd1ac020a4160289b6accaf1108239631e

**Reply by GhostWalker562** [reply]
If your wallet was rotated without you knowing, it is likely your wallet is compromised and is unrecoverable.

**Reply by adeshkhari** [reply]
Pls help 🙏

**Comment by adeshkhari** [comment]
Brother where are you from

**Comment by tbuyuksahin** [comment]
"Please help me recover this wallet: 0x7ac7a8224a2e935e5847983215150f62471a6c954d5ab82b8134e849c36c388c. I'm getting the INVALID_AUTH_KEY error."

**Reply by adeshkhari** [reply]
Same problem brother my wallet invalid auth key

**Reply by Johnivan113** [reply]
same problem. Account was rotated without my participation. How can I fix it?0x6af15e4ffdeeb8504573c02f673b26369c71ecf5302dd26332ed9a2f0cd999ca
